gulpfile.js

```diff
@@ -24,10 +26,13 @@ exports.scripts = series(cleanJs, scripts);
 exports.styles = series(cleanCss, styles);
 exports.watch = watch;
 exports.cc = createComponent;
+exports.svg = svg;
+exports.iconfont = iconfont;
```

Comment:
maybe should be also that task:
icons: `parallel(svg, iconfont),`

Reply:
why can't we just call it on `gulp` default task ?

Reply:
fixed